### ISSUE TITLE
Image based marker

### ISSUE THREAD

**Issue by vikssin17** [jeromeetienne/AR.js#464]
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
feature not working
**If the current behavior is a bug, please provide the steps to reproduce.**
not a bug 

**What is the expected behavior?**
I want image based marker to be worked
**If this is a feature request, what is motivation or use case for changing the behavior?**
i want it for image based marker should be supported by AR.js
**Please mention other relevant information such as the browser version, Operating System and Device Name**
working fine on all browsers

I want to know is image based marker supported by AR.js

**Comment by nicolocarpignoli** [comment]
Hi, it is not depending on AR.js but rather on artoolkit https://github.com/artoolkit
AR.js is 'just' a great library that includes several libraries to make marker-based-web-AR accessible and easy for everyone. What you ask is out of this library scope. Take a look at the artoolkit repositories to learn more about markers. Thanks

**Comment by matthias77** [comment]
Hi,
 I found an example working with aframe-ar.js. I tested it with an image (normal marker creation but with an colored image) and voila - it works nice. I tried to reproduce it with ar.js and three.js and it won't working, don't know why. I think this must be an configuration or somthing like this. I hope someone can explain that to me. I'm impressed how fast and stable ar.js works, but I really think the usage of nearly "normal" images improves the capabilities and make it much more sexy and would a real alternative to markerless AR.

Many thanks, matthias
